cypress/e2e/api.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('EngageSphere - API', () => {
+  it('returns the correct status and body structure', () => {
+    cy.api({
+      method: 'GET',
+      url: `${Cypress.config('apiUrl')}/customers`
+    }).then( response => {
+      expect(response.status).to.eq(200)
+      expect(response.body).to.have.property('customers')
+      expect(response.body).to.have.property('pageInfo')
+    })
+  })
+})
```

Comment:
```suggestion
})

```

É recomendado deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como o GitHub não exibem o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção que deve ser seguida.

Reply:
O mesmo vale para outros arquivos.